index.bs

```diff
@@ -65,6 +66,8 @@ spec: ecma262; urlPrefix: https://tc39.github.io/ecma262/
 spec: storage; urlPrefix: https://storage.spec.whatwg.org/
     type: dfn
         text: storage bucket; url: storage-bucket
+        text: storage key; url: storage-key
```

Comment:
Do we want to call this out as temporary anywhere? Or is there a plan for these to be exported from storage?

Reply:
I'm not sure what the plan is @annevk

Reply:
Yeah, that's https://github.com/w3c/IndexedDB/pull/334 which got complicated and stalled since as written it _also_ relied on non-exported concepts. I'll need to stare at it for a while to remember how far along that was.